guide/getting-started/14-hardware.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: guide
+title: Hardware overview
+nav_order: 14
+parent: Getting started
+permalink: /getting-started/hardware/
+---
+
+<!--
+
+Editor's notes
+
+This page provides an overview of the different types of hardware that interact with the bitcoin network.
+
+-->
+
+# Hardware overview
+
+Generally, any hardware with an internet connection can interact with the bitcoin network. While our personal computers and smartphones work great for general wallet management and transactions, a few types of dedicated hardware have been created that are optimized for the more unique aspects of how bitcoin works.
+
+## Recovery phrase storage
+
+Simple, cheap and robust storage solutions for recovery phrases come in modest forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, in comparison, is the size of a credit card and comes pre-loaded with a recovery phrase (hidden behind a security seal) and a matching bitcoin address. And if you like an even simpler solution, a piece of paper will do. Although paper is not as durable and may be easy to loose, it is one of the strengths of Bitcoin that such simple solutions are possible.
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/hardware/hardware-seed-storage.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/hardware-seed-storage@2x.jpg"
+   alt-text = "Examples of seed storage options"
+   width = 800
+   height = 400
+%}
+
+## Hardware wallets
+
+You may already be familar with key fobs from your bank or work. They are small devices with specific built-in authentication that is used to control and secure access to a computer system. Key fobs are not connected to the internet or any other network, so they cannot easily be hacked without physical access.
```

Comment:
While I like and understand this example, people in third-world countries like myself may not have an idea of what a key fob is. Don't have a personal preference, just want to point out that if our goal is worldwide audience, we probably should use something a bit more generic, though I can't think of anything atm, so just consider this as a suggestion for the future.

Reply:
👋 third-world country guy here too.

Never heard of key fobs, had to google 😅